tests/baselines/reference/JSDocParsing/DocComments.parsesCorrectly.noReturnType.json

```diff
@@ -6,7 +6,7 @@
         "0": {
             "kind": "JSDocReturnTag",
             "pos": 8,
-            "end": 16,
+            "end": 15,
```

Comment:
Odd to see all these end position shift sometimes +2, sometime +1, and sometimes -1. I assume you've done a spot check and the new positions are (hopefully more) correct in each case?

Reply:
Yeah, the end just no longer includes the trailing whitespace at the end of the comment.